### ISSUE TITLE
Is there a way to disable pinch zoom?

### ISSUE THREAD

**Issue by drahnieR** [qvacua/vimr#776]
If working on mbp, touchpad could be mis-touched and ends up with an unintentional zoom, which is quite annoying.

**Comment by unnamed777** [comment]
I'd like to have this option too, it's very annoying.

**Comment by cameroj** [comment]
Agree. Part of the appeal of vim is never moving your hands from the keyboard. This seems contrary to that philosophy. I configure my font size as I like it, and would prefer it never change. Adjusting font size using pinch zoom is unpredictable and imprecise (and probably unintentional).
